exercises/intro/intro1.d

```diff
@@ -1,3 +1,14 @@
+/// Sometimes just solving an exercise isn't enough.
+/// We encourage you stay and mess around with the exercises, until you feel
+/// confortable to move forward.
+/// When you feel you're done with an exercise remove the 'Just D it' comment below.
+
+/// You can get a hint by either running `dlings hint intro1` or by typing `hint`
+/// in watch mode.
+/// If you're in watch mode the exercise will be recompiled each time you modify
+/// the file! Try messing around with it a bit, delete or add some `writeln`,
```

Comment:
I'd remove the exclamation here, since it's used later on.
```suggestion
/// the file. Try messing around with it a bit, delete or add some `writeln`,
```

Reply:
Fixed in 340875012539db8cdae6845c66a9bb6ee1b64751